core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -29,15 +30,16 @@ public NoPlayerView(Context context, AttributeSet attrs) {
     public NoPlayerView(Context context, AttributeSet attrs, int defStyleAttr) {
         super(context, attrs, defStyleAttr);
         surfaceHolderProvider = new PlayerViewSurfaceHolder();
-        aspectRatioChangeListener = new AspectRatioChangeListener(this);
+        aspectRatioChangeListener = new AspectRatioChangeForwarder(this);
     }
 
     @Override
     protected void onFinishInflate() {
         super.onFinishInflate();
         View.inflate(getContext(), R.layout.noplayer_view, this);
-        videoFrame = Views.findById(this, R.id.video_frame);
+        videoFrame = Views.findById(this, R.id.exo_content_frame);
```

Comment:
This is internal to `SimpleExoPlayerView` which is a `FrameLayout` that performs some operations on an `ExoPlayer` however, this is a `FrameLayout` whose main purpose is to expose things like a "Controller", "SubtitlesView" and "SurfaceView" (for rendering video). We reasoned that we could reuse this view for both the `MediaPlayer` and `ExoPlayer` because in terms of the `MediaPlayer` we only care for the internal `SurfaceView` and all other methods are ignored.